### DISCUSSION TITLE
PCB Through Hole Size

### DISCUSSION THREAD

**Discussion by c64manguy** [frntc/SIDKick-pico#60] [General]
Can you tell me what size through holes you used for the Rpi Pico and the header pins? I am trying to put together a pin/socket setup that let's me mount the pi pico with an incredibly low profile and still be socketed.

**Comment by frntc** [comment]
(I convert this to a discussion, it's not an "issue".)

The hole diameter is 0.8mm.

**Reply by c64manguy** [reply]
Thank you so much for all of the great projects! If you need any NTSC stuff tested I'd be happy to help. I have several 250407 boards, a couple 250425s, a 250466, and a 326298 rev. A PCB. I have an NTSC C128 as well. And so far I have the Sidkick pico rev 1 and the Sidkick Pico rev 2.
I really appreciate what you're doing for the community.
